src/layers/recurrent.jl

```diff
@@ -173,12 +211,17 @@ end
 
 @layer RNN
 
+initialstates(rnn::RNN) = zeros_like(rnn.cell.Wh, size(rnn.cell.Wh, 1))
```

Comment:
```suggestion
initialstates(rnn::RNN) = initialstates(rnn.cell)
```
and similarly for the other recurrent layers